doc/lsst.obs.decam/using-obs-decam.rst

```diff
@@ -0,0 +1,79 @@
+.. _lsst.obs.decam.gen3:
+
+#################
+The Gen3 workflow
+#################
+
+These directions are for users who wish to create their own data repositories independent of any Rubin/LSST-supported shared repositories.
+
+Create a Gen3 data repository and write curated calibration products to it (these include crosstalk, linearity, and defects). The latter step creates the `DECam/calib` collection.
+
+.. prompt:: bash
+
+   cd /path/to/desired/repo
```

Comment:
For anything with a path like this, it's probably better to have `set BUTLER_PATH=/path/to/repo` as the first step, and then refer to `$BUTLER_PATH` everywhere below (instead of `.` in this example), so that the user isn't running all of the commands directly in the given directory (so that, for example, log files or redirected output don't get written to the butler directory).

Especially for gen3, users should not be touching or writing files in the butler repo itself (since we now have a working S3 backend, for example).

Reply:
haha, I think you left 2 versions of the same comment twice, or else I just assumed this bit was also what you meant 😀